doc/configuration/privacyidea-appliance.rst

```diff
@@ -112,14 +112,54 @@ Restore
 The entry `View Backups` will list all the backups available.
 
 .. figure:: images/appliance/backup3.png
-   :scale: 50%
+   :width: 550px
 
    All available backups
 
 You can select a backup and you are asked if you want to restore the data. 
 
 .. warning:: Existing data is overwritten and will be lost.
 
+.. _database:
+
+Database: Setup Redundancy
+..........................
+
+The appliance-tool is also capable of setting up a redundant setup between
+two privacyIDEA nodes. For the setup, the privacyIDEA slave instance has to
+be setup and running with identical software than the master instance. The
+appliance-tool can use a tinc-tunnel to share the configuration and the database.
+
+.. figure:: images/appliance/database.png
+   :width: 400px
+
+.. note:: If you choose to use the tinc VPN connection between
+   the nodes and an SSH root login make sure the services are installed.
+
+.. warning:: Existing data on the second node is overwritten and will be lost.
+
+.. figure:: images/appliance/redundancy-successful.png
+   :width: 450px
+
+.. _pi-appliance_updates:
+
+Updates
+.......
+
+In this menu, you can setup cronjobs for automatic updates which is seldom
+used in productive setups.
+
+.. _pi-appliance_audit:
+
+Audit Rotation
+..............
+
+In the `Audit Rotation` menu, you can setup cronjobs for the audit rotation conditioned
+by age or the number of entries. The syntax follows the crontab syntax as explained
+in :ref:`backup_and_restore`.
+
+.. note:: Keep in mind that the audit log is synchronized between the nodes in a redundant
+   setup. Therefore an audit log rotation is only necessary on one node.
```

Comment:
Therefore you should not run an audit rotation on two nodes at the same time!

Reply:
ok